pgjdbc/src/main/java/org/postgresql/util/CopyNotSupportedException.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2019, PostgreSQL Global Development Group
+ * See the LICENSE file in the project root for more information.
+ */
+
+
+package org.postgresql.util;
+
+public class CopyNotSupportedException extends PSQLException {
```

Comment:
exceptions are serializable. should we generate a serial version id?
It looks like PSQLException does not have one declared either...